### DISCUSSION TITLE
[keyball61]右手の縦一列が入力出来ない

### DISCUSSION THREAD

**Discussion by dragonnizum** [Yowkees/keyball#467] [Q&A]
Keyball61の右手側入力できない縦一列があり、半田やり直しなどしてますが、原因特定に至ってません。
写真の養生が貼ってある列となります。
また、￥マークのキーについては、テストで反応するもテキストエディタ等では入力されない状態であり、こちらについてもアドバイスいただきたく。
![IMG_2516](https://github.com/Yowkees/keyball/assets/155537375/aecda38f-984d-44a6-93df-ddd84d9ae0e8)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

反応がない縦一列のキーはCOL0という信号線でつながっていますので、ProMicro部分にありますCOL0と書かれた端子のはんだ付けをやり直していただけますと幸いです。

隣の一つだけ効かないキーはすぐ上のダイオードが対応している物になりますので、キーソケットのはんだ付けと併せてやり直しをお願いいたします。

**Reply by dragonnizum** [reply]
Ryo624様

お返事ありがとうございます。

再度ピコマイクロ周辺の半田付け見直してみます。
隣の1つのみ反応しないキーについてですが、Remapのテストでは押したタイミングで反応しているのですが、テキストエディタでの入力が確認できない状態です。
Remapテストでは反応しているのでダイオードの可能性は低いのではないかと考えているのですが…
単純なキーアサインの勘違いもあるかもしれませんので、もう少し調べてみますが、何か思いつくことがあれば、ご助言いただけると幸いです。
